installscripts/cookbooks/jenkins/files/default/jazz-installer-vars.json

```diff
@@ -75,7 +75,10 @@
             "API": "build-pack-api",
             "LAMBDA": "build-pack-lambda",
             "WEBSITE": "build-pack-website"
-        }
+        },
+        "LAMBDA_SERVICES": ["cloud-logs-streamer", "deployments-event-handler", "environment-event-handler"],
+        "API_SERVICES": ["acl", "create-serverless-service", "environment-event-handler", "delete-serverless-service"],
```

Comment:
`environment-event-handler` is not an API. And, it's added to the previous list. Please remove it from here.

Reply:
Also, I see that  few APIs are missing here (that needs access to these components that  are in VPC now) -
- jazz_codeq (talk to sonar)
- jazz_logs (talks to ES)